sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousExecution.scala

```diff
@@ -391,6 +393,7 @@ class ContinuousExecution(
 }
 
 object ContinuousExecution {
+  val IS_CONTINUOUS_PROCESSING = "__is_continuous_processing"
```

Comment:
nit: I think this belongs in StreamExecution, since both ContinuousExecution and MicroBatchExecution set it.